carla/models/catalog/catalog.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+
+from ..api import MLModel
+from .load_model import load_model
+
+
+class MLModelCatalog(MLModel):
+    def __init__(
+        self, data_name, model_type, ext="h5", cache=True, models_home=None, **kws
+    ):
+        self._data_name = data_name
+        self._model = load_model(model_type, data_name, ext, cache, models_home, **kws)
+
+        if ext == "pt":
+            self._backend = "pytorch"
+        elif ext == "h5":
+            self._backend = "tensorflow"
+        else:
+            raise Exception("Model type not in catalog")
+
+        self._name = model_type + "_" + data_name
```

Comment:
Are we using the name somewhere? If not lets remove it and lets only add source code if/once we need it

Reply:
Alright, will remove it.